master/getting-started/kubernetes/installation/other.md

```diff
@@ -31,35 +31,31 @@ complete the following steps.
 
 1. If your cluster contains more than 50 nodes:
```

Comment:
Just to clarify: Typha is enabled by default. This is a yaml update that happened some time ago, but the docs did not reflect this change yet.